### PR TITLE
Dev/better-xterm-logging

### PR DESCRIPTION
**Pull request by iliajie**
Hey Jamie!

This PR implements a small improvements to how logging is done in Terminal module.

### CODE REVIEW

File: xterm/index.cgi [hunk 1]
```diff
@@ -194,11 +194,10 @@ my $shellserver_cmd = "$module_config_directory/shellserver.pl";
 if (!-r $shellserver_cmd) {
 	&create_wrapper($shellserver_cmd, $module_name, "shellserver.pl");
 	}
-my $tmpdir = &tempname_dir();
 $ENV{'SESSION_ID'} = $main::session_id;
 &system_logged($shellserver_cmd." ".quotemeta($port)." ".quotemeta($user).
 	       ($dir ? " ".quotemeta($dir) : "").
-	       " >$tmpdir/ws-$port.out 2>&1 </dev/null");
+	       " >$module_var_directory/websocket-connection-$port.out 2>&1 </dev/null");
```
Comment (jcameron) [inline xterm/index.cgi:200]: Is this change related to logging?

Comment (iliajie) [reply xterm/index.cgi:200]: Yes, it will write logs to the right place!

Comment (jcameron) [reply xterm/index.cgi:200]: Ok makes sense!
